wordpress-mobile/WordPress-iOS
# [Domain Focus] Card UI

This PR adds the UI for Domain Focus Card.

### TODO for following PR's: 
- [ ] Button functionality
- [ ] Tracks

### To test:
1. Install and Launch Jetpack App.
2. Enable the "Domain Focus" Feature Flag.
3. Verify if the "Google Domains" card is displayed on My Site.
4. Verify the location of the card and the UI.
5. Test in Landscape mode & and both light/dark modes.

### Screenshots
| Light Mode  | Dark Mode |
| ------------- | ------------- |
| ![Simulator Screenshot - iPhone 14 Pro - 2023-08-03 at 14 55 24](https://github.com/wordpress-mobile/WordPress-iOS/assets/15968946/545d47b9-6d81-4f75-8b1f-489b3c893990) | ![Simulator Screenshot - iPhone 14 Pro - 2023-08-03 at 14 55 27](https://github.com/wordpress-mobile/WordPress-iOS/assets/15968946/702a7169-c3c9-4664-9aaf-3c0151db82ae) |

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
